nullaway/src/main/java/com/uber/nullaway/handlers/Handler.java

```diff
@@ -314,6 +316,19 @@ void onDataflowVisitLambdaResultExpression(
    */
   ImmutableSet<String> onRegisterImmutableTypes();
 
+  /**
+   * Called when a method writes a {@code @Nonnull} value to a class field. Since traversing AST is
+   * costly, we do it only inside the handler when the feature is enabled and, this method accesses
+   * the initializer method through the leaf node in state parameter.
```

Comment:
Seems like some of these comments are specific to the handler you wrote as part of this PR?  In principle, future handlers that implement this method may do something different.  I'd move the text after `Since` to your new handler

Reply:
63ce1c712bef9f5927496ba8e7075f2e3ea0a188